### PR TITLE
Crud deno

### PR DESCRIPTION
**Pull request by rajeevkilari**

### CODE REVIEW

File: src/main/java/com/example/demo/interfaces/StudentInterface.java [hunk 1]
```diff
@@ -11,10 +12,20 @@
...
+
+	List<StudentEntity> findBySchoolName(String schoolName);
+
+	@Query(value = "select * from Student_info s where s.name=:name", nativeQuery = true)
+	List<StudentEntity> getName(String name);
+
+	@Query(value = "select * from Student_info s where s.section=:section", nativeQuery = true)
+	List<StudentEntity> getSection(String section);
+
+	@Query(value = "update Student_info s set s.name='nitesh' where s.section=:section", nativeQuery = true)
```
Comment (codetalks95) [inline src/main/java/com/example/demo/interfaces/StudentInterface.java]: Is this working Good

Comment (rajeevkilari) [reply src/main/java/com/example/demo/interfaces/StudentInterface.java]: getSection is working fine

File: src/main/java/com/example/demo/repository/StudentRepository.java [hunk 2]
```diff
@@ -1,139 +1,174 @@
...
+		} else {
+			response.setMessage("The Data is Present with this Name" + " " + name);
+			response.setStatusCode(HttpStatus.FOUND);
+			response.setStudentEntityList(studentEntityList);
+		}
+		response.setDateTime(Utility.getDate());
+		return response;
+	}
 
+<<<<<<< HEAD
```
Comment (codetalks95) [inline src/main/java/com/example/demo/repository/StudentRepository.java]: Fix this  <<<<<<< HEAD should not be present

File: src/main/java/com/example/demo/repository/StudentRepository.java [hunk 3]
```diff
@@ -1,139 +1,174 @@
...
+			response.setStatusCode(HttpStatus.NOT_FOUND);
+		} else {
+			response.setMessage("The Data is Present with this Section" + " " + section);
+			response.setStatusCode(HttpStatus.FOUND);
+			response.setStudentEntityList(studentEntityList);
+		}
+		response.setDateTime(Utility.getDate());
+		return response;
+	}
+=======
```
Comment (codetalks95) [inline src/main/java/com/example/demo/repository/StudentRepository.java]: Remove this

File: src/main/java/com/example/demo/repository/StudentRepository.java [hunk 4]
```diff
@@ -149,57 +184,80 @@ public Response findByName(String name) {
         response.setDateTime(Utility.getDate());
         return response;
     }
+>>>>>>> 69830039cae3d34fcec997a54569ad312a0a8aa2
```
Comment (codetalks95) [inline src/main/java/com/example/demo/repository/StudentRepository.java]: Remove this

File: src/main/java/com/example/demo/interfaces/StudentInterface.java (hunk 1 above)
Comment (rajeevkilari) [reply src/main/java/com/example/demo/interfaces/StudentInterface.java]: updateName is not working

**Review by codetalks95 — CHANGES_REQUESTED** [review]
Please do the essential changes requsted

**Review by codetalks95 — APPROVED** [review]

**Pull request merged**